File: 2nd Semester/C Programming/exp-02.c
//Write a Program in C to find the average of n numbers by using for loop

#include<stdio.h>
int main()
{
    int i,n,sum=0;
    printf("Enter the number of elements: ");
    scanf("%d",&n);
    for(i=1;i<=n;i++)
    {
        printf("Enter the %d element: ",i);
        scanf("%d",&sum);
    }
    printf("The average is: %d",sum/n);
    return 0;
}

File: 3rd Semester/OOPS using JAVA/exp-02/exp_02.java
//Write a Java program to print the sum of two numbers.

import java.util.Scanner;
 public class exp_02 {
    public static void main(String[] args) {
          int first_num, second_num, sum;
            Scanner input = new Scanner(System.in);
            System.out.print("Enter first number: ");
            first_num = input.nextInt();
            System.out.print("Enter second number: ");
            second_num = input.nextInt();
            input.close();
            sum = first_num + second_num;
            System.out.println("Sum of " + first_num + " and " + second_num + " is " + sum);
    }
}
